specs/fixture.automation.md

```diff
@@ -2,7 +2,7 @@
 testspace:
 title: Fixture Automation
 before:
-  name: github::setup
+  name: github::fn-name1
```

Comment:
Further down:
* `input` should be `payload`
* `**SETUP FIXTURE**` should be `**BEFORE FIXTURE**`

Reply:
updated